utils/jsonhex/types.go

```diff
@@ -44,8 +44,8 @@ func (h *Bytes) UnmarshalJSON(data []byte) error {
 
 // String returns the hex string representation of Bytes.
 func (h Bytes) String() string {
-	if h == nil {
-		return `"null"`
+	if h == nil || len(h) == 0 {
```

Comment:
Please consider following the suggestion made by GitHub Actions (here and in other places).